android/app/src/main/java/org/pathcheck/covidsafepaths/exposurenotifications/dto/RNExposureInformation.java

```diff
@@ -5,10 +5,10 @@
 @SuppressWarnings({"FieldCanBeLocal", "unused"})
 public class RNExposureInformation {
   private String id;
-  private long date;
-  private long duration; // Minutes
+  private long date; // Milliseconds
```

Comment:
It's called `date` on the JS side and we have to maintain the same naming because it will be parsed from a JSON string.
In a future PR I rename it to "posix" here and add an annotation to write it as "date" in the JSON we are sending to the JS layer

```
export interface RawExposure {
  id: UUID
  date: Posix
  duration: number
}
```